pom.xml

```diff
@@ -19,8 +19,8 @@
   <properties>
     <java.version>1.8</java.version>
     <avro.version>1.8.0</avro.version>
-    <!-- informative only, used in about template substitution -->
-    <scala.version>2.10</scala.version>
+    <scala.version>2.10.6</scala.version>
+    <scala.version.prefix>2.10</scala.version.prefix>
```

Comment:
I think it is OK (preferable, really) to have `About` print the full Scala version, as opposed to the prefix.